cdap-ui/app/wrangler/components/Wrangler/WrangleData/index.js

```diff
@@ -87,6 +96,89 @@ export default class WrangleData extends Component {
     WranglerStore.dispatch({ type: WranglerActions.redo });
   }
 
+  generateLinks() {
+    let wranglerPluginProperties = this.props.onHydratorApply();
+
+    let batchArtifact = {
```

Comment:
Yeah atleast we can remove the hardcoding of versions and scope. Thats usually the one that changes across releases.